neqo-bin/src/bin/client/http3.rs

```diff
@@ -255,8 +255,10 @@ impl StreamHandlerType {
         match handler_type {
             Self::Download => {
                 let out_file = get_output_file(url, &args.output_dir, all_paths);
+                let buf_writer =
+                    out_file.map(|file| BufWriter::with_capacity(BUFWRITER_BUFFER_SIZE, file));
```

Comment:
How about moving this into `get_output_file`. `get_output_file` is called by both `http3` and `http09`, thus no need for duplication.